redis-cache.tf

```diff
@@ -39,20 +39,8 @@ resource "azurerm_redis_firewall_rule" "container_app_default_static_ip" {
   name                = "${replace(local.resource_prefix, "-", "")}containerapp"
   redis_cache_name    = azurerm_redis_cache.default[0].name
   resource_group_name = local.resource_group.name
-  start_ip            = azurerm_container_app.container_apps["main"].outbound_ip_addresses[0]
-  end_ip              = azurerm_container_app.container_apps["main"].outbound_ip_addresses[0]
-}
-
-resource "azurerm_redis_firewall_rule" "container_app_worker_static_ip" {
```

Comment:
This is redundant because all the containers use the same outgoing IP address (the PIP assigned to the K8s LB)